wagtail/snippets/templates/wagtailsnippets/snippets/create.html

```diff
@@ -16,14 +16,8 @@
         </div>
 
         <footer class="footer">
-            <nav aria-label="{% trans 'Actions' %}">
-                <ul>
-                    <li class="actions actions--primary footer__container">
-                        <div class="dropdown dropup dropdown-button match-width">
-                            {{ action_menu.render_html }}
-                        </div>
-                    </li>
-                </ul>
+            <nav class="actions actions--primary footer__container" aria-label="{% trans 'Actions' %}">
```

Comment:
This doesn't have the `w-flex` and `w-flex-grow` combo and results in incorrect width if there's only one action, e.g. if the user only has "add"/"change" permission (not delete/unpublish), e.g.

<img width="345" alt="image" src="https://github.com/wagtail/wagtail/assets/6379424/353b6d59-05d1-4af1-844a-008bbdb9d9bd">

Reply:
Thank you, I’ll fix that one with `w-grid` at the expense of consistency so we only have to change those two templates.